<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>40</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
      <widget class="QLabel" name="serverNameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverNameLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="serverPortLabel">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>150</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port:</string>
    </property>
   </widget>
    <widget class="QLineEdit" name="serverPortLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createRoomBtn">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>140</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Create Room</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinRoomBtn">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>190</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Join Room</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leaveBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
   </widget>
  <widget class="QLabel" name="playerCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverNameDetailLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverPortDetailLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="playerList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <zorder>nameLineEdit</zorder>
   <zorder>createRoomBtn</zorder>
   <zorder>joinRoomBtn</zorder>
   <zorder>createBtn</zorder>
   <zorder>joinBtn</zorder>
   <zorder>backBtn</zorder>
   <zorder>startBtn</zorder>
   <zorder>leaveBtn</zorder>
   <zorder>serverNameLineEdit</zorder>
   <zorder>serverPortLabel</zorder>
   <zorder>serverNameLabel</zorder>
   <zorder>nameLabel</zorder>
   <zorder>serverPortLineEdit</zorder>
   <zorder>playerCountLabel</zorder>
   <zorder>serverNameDetailLabel</zorder>
   <zorder>serverPortDetailLabel</zorder>
   <zorder>playerList</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
